src/components/dialog/dialog.js

```diff
@@ -588,9 +590,12 @@ function MdDialogProvider($$interimElementProvider) {
       options.unlockScreenReader();
       options.hideBackdrop(options.$destroy);
 
+      // Remove the focus traps that we added earlier for keeping focus within the dialog.
+      topFocusTrap.parentNode.removeChild(topFocusTrap);
```

Comment:
Each of the focus traps has different parents. The top is a sibling to the dialog element, the bottom is a child of it.